app/controllers/declaration/RoutingCountriesSummaryController.scala

```diff
@@ -46,18 +46,13 @@ class RoutingCountriesSummaryController @Inject()(
     extends FrontendController(mcc) with I18nSupport with ModelCacheable {
 
   def displayPage(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
-    request.declarationType match {
-      case DeclarationType.SUPPLEMENTARY =>
-        navigator.continueTo(controllers.declaration.routes.DestinationCountryController.displayPage(mode))
-      case _ =>
-        val countryCodes = request.cacheModel.locations.routingCountries
-        val countries = findByCodes(countryCodes)
-
-        if (countries.nonEmpty) {
-          Ok(routingCountriesSummaryPage(mode, RoutingQuestionYesNo.form(), countries))
-        } else {
-          navigator.continueTo(controllers.declaration.routes.RoutingCountriesController.displayRoutingQuestion(mode))
-        }
+    val countryCodes = request.cacheModel.locations.routingCountries
```

Comment:
What about case that you removed? This screen is not a part of the `SUPPLEMENTARY` journey